Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,36 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen;
+
+    assert(ends != NULL);
+    assert(count != NULL);
+    numopen = send ? ends->numsendopen : ends->numrecvopen;
```

Comment:
```suggestion
    int64_t numopen = send ? ends->numsendopen : ends->numrecvopen;
```

Hmm, why not use `count` directly?

```suggestion
    *count = send ? ends->numsendopen : ends->numrecvopen;
```

Reply:
Here I was following the principle that OUT args should not be modified in the case of an error occurring (which it's technically possible could happen in this function).

Reply:
I see what you are saying.  I guess I haven't ever worried about it in functions where the OUT arg is only used in the success case. :)